test/e2e/backup_test.go

```diff
@@ -161,7 +161,7 @@ func testEtcdBackupOperatorForS3Backup(t *testing.T, clusterName, operatorClient
 		if err != nil {
 			return false, fmt.Errorf("failed to retrieve backup CR: %v", err)
 		}
-		if reb.Status.Succeeded {
```

Comment:
This check doesn't seem right.
We should not keep looping if result is returned. If returned result is not expected, we should fail:
```
t.Errorf("exp revision=xxx, get=yyy")
```

Reply:
fixed!